pkg/k8s/node.go

```diff
@@ -65,6 +67,13 @@ func ParseNode(k8sNode *slim_corev1.Node, source source.Source) *nodeTypes.Node
 			continue
 		}
 
+		encountered = addressTypes[addr.Type]
+		if encountered {
+			scopedLog.Errorf("address type %s is repeated for node %s", addr.Type, k8sNode.Name)
+		} else {
+			addressTypes[addr.Type] = true
+		}
```

Comment:
I think what might be better here is to say something like we've detected multiple IPs of the same type and Cilium is going to only consider the first / last one. I'm not sure which one the logic will select, i.e. either the first or last one.